src/api/interactivity.js

```diff
@@ -3,6 +3,66 @@ import Layer from './layer';
 import { WM_R, projectToWebMercator } from './util';
 import { wToR } from '../client/rsys';
 
+
+/**
+ * 
+ * FeatureEvent objects are fired by {@link carto.Interactivity|Interactivity} objects.
+ * @typedef {Object} FeatureEvent
+ * @property {object} coordinates LongLat coordinates in {lng: 0, lat:0} form
```

Comment:
missing id property

Reply:
Missing on `Feature`, not here, and that's not implemented on `master`